packages/webpack-cli/lib/utils/logger.js

```diff
@@ -1,7 +1,13 @@
 const { red, cyan, yellow, green } = require('colorette');
 
 module.exports = {
-    error: (val) => console.error(`[webpack-cli] ${red(val)}`),
+    error: (val) => {
+        if (val && val.stack) {
+            val = val.stack;
+        }
+
+        console.error(`[webpack-cli] ${red(val)}`);
```

Comment:
Why? error.toString() always show the stacktrace

Reply:
In Node 14 `error.toString()` only shows the error message.

Reply:
Ditto in Node 12.x and 13.x as well.

Reply:
https://github.com/webpack/webpack-cli/pull/1940#discussion_r504598344